app.rb

```diff
@@ -0,0 +1,27 @@
+require 'sinatra'
+require 'jwt'
+
+METABASE_EMBEDDING_SECRET = ENV.fetch('METABASE_EMBEDDING_SECRET')
+METABASE_SITE_URL = ENV.fetch('METABASE_SITE_URL') || 'http://localhost:3000'
+METABASE_EMBED_DASHBOARD_ID= ENV.fetch('METABASE_EMBED_DASHBOARD_ID')
+
+get '/' do
+  payload = {
+    :resource => {:dashboard => 8},
+    :params => {          
+    },
+    :exp => Time.now.to_i + (60 * 10) # 10 minute expiration
+  }
```

Comment:
you are going to use symbols in a hash, the style is not to use the =>.  So more like
```
payload = {
    resource: { dashboard: 8},
    params:  {},
    exp: Time.now.to_i + (60 * 10) # 10 minute expiration
  }
```

Reply:
Updated.